<launch>
    <node
        pkg="jsk_spot_delivery_demo"
        type="pickup_package_client.py"
        name="pickup_package_client"
        output="screen">
    </node>
</launch>
